README-Mandrel.md

```diff
@@ -0,0 +1,52 @@
+# Mandrel
+
+Mandrel is a downstream distribution of the GraalVM community edition.
+Its main goal is to provide a `native-image` in-place replacement to be used by [Quarkus](https://quarkus.io).
+
+## How Does Mandrel Differ From Graal
+
+Mandrel releases are built from a code base derived from the upstream GraalVM code base, with only minor changes but some significant exclusions. 
+They support the same native image capability as GraalVM with no significant changes to functionality.
+They do not include support for Polyglot programming via the Truffle interpreter and compiler framework. 
+In consequence, it is not possible to extend Mandrel by downloading languages from the Truffle language catalogue.
+
+Mandrel is also built slightly differently to GraalVM, using the standard OpenJDK project release of jdk11u.
+This means it does not profit from a few small enhancements that Oracle have added to the version of OpenJDK used to build their own GraalVM downloads.
+Most of these enhancements are to the JVMCI module that allows the Graal compiler to be run inside OpenJDK.
+The others are small cosmetic changes to behaviour.
+These enhancements may in some cases cause minor differences in the progress of native image generation.
+They should not cause the resulting images themselves to execute in a noticeably different manner.
+
+## Communication Channels
+
+* [Slack](https://graalvm.slack.com) - Join #mandrel channel
+* [graalvm-dev@oss.oracle.com](mailto:graalvm-dev@oss.oracle.com?subject=[MANDREL]) mailing list - Subscribe [here](https://oss.oracle.com/mailman/listinfo/graalvm-dev)
+* [GitHub issues](https://github.com/graalvm/mandrel/issues) for bug reports, questions, or requests for enhancements.
+
+Please report security vulnerabilities according to the [Reporting Vulnerabilities guide](https://www.oracle.com/corporate/security-practices/assurance/vulnerability/reporting.html).
+
+## Getting Started
+
+Mandrel distributions can be downloaded from [the repository's releases](https://github.com/graalvm/mandrel/releases)
+
+### Prerequisites
+
+Mandrel's `native-image` depends on the following packages:
+* glibc-devel
+* zlib-devel
+* gcc
+* libffi-devel
+
+On Fedora/CentOS/RHEL they can be installed with:
+```bash
+dnf install glibc-devel zlib-devel gcc libffi-devel
+```
+
+### Building Mandrel From Source
+
+For building Mandrel from source please see [mandrel-packaging](https://github.com/graalvm/mandrel-packaging)
```

Comment:
@galderz @zakkak This should eventually become true, but it is not ATTOW.  Let's leave it here. Someone has to do the migration though :)

Reply:
Started discussion in https://oss.oracle.com/pipermail/graalvm-dev/2020-June/000061.html